xmodule/course_block.py

```diff
@@ -296,6 +297,22 @@ def get_available_providers():  # lint-amnesty, pylint: disable=missing-function
     return available_providers
 
 
+def get_requires_escalation_email_providers():
+    """Return list of available proctoring providers that require an escalation email."""
```

Comment:
```suggestion
def get_requires_escalation_email_providers() -> list[str]:
    """
    Return list of available proctoring providers that require an escalation email.
    """
```
formatting & type annotation

Reply:
@mlabeeb03 if you can just apply this suggestion and the other one, then I'll go ahead and merge